public/sections/todo/todo.html

```diff
@@ -4,19 +4,23 @@
         <input type="text" placeholder="Search" ng-model="findTodo">
     </li>
     <li class="collection-item" ng-repeat="todo in ctrl.todos | filter : findTodo">
-        <a><i class="material-icons waves-circle tiny padding-circle" style="background-color : {{todo.color}}; ">circle</i></a>
+        <a href="" style="background-color : {{todo.color}}; " class="material-icons tiny btn-floating"></a>
         <strong> {{todo.name}}</strong>
-        <a href="" class="secondary-content padding-edit-button" ><i class="material-icons">edit</i></a>
+        <a href="" class="secondary-content padding-edit-button"><i class="material-icons">edit</i></a>
```

Comment:
We should probably avoid empty `href` attributes, something like `href="javascript:void(0);` should work fine